crates/inspector/src/traits.rs

```diff
@@ -18,28 +18,62 @@ use crate::{
 /// It is used inside [`crate::InspectorHandler`] to extend evm with support for inspection.
 pub trait InspectorEvmTr:
     EvmTr<
-    Frame = EthFrame<EthInterpreter>,
+    Frame: InspectorFrameTr<IT = EthInterpreter>,
```

Comment:
Removes dependency on `EthFrame` and introduced `InspectorFrameTr` that has `fn eth_frame() -> Option<&EthFrame>`